BOOTSTRAP.adoc

```diff
@@ -97,6 +97,17 @@ There are five steps to renaming a primitive:
 It is desirable for bootstraps to be easily repeatable, so you should commit
 changes after step 4.
 
+To remove a primitive:
+1. Start with a working build of the compiler eg `./configure && make`
```

Comment:
```suggestion
1. Start with a working build of the compiler eg `./configure && make world`
```
`make world` should suffice for a bootstrap.

Reply:
Updated. Thanks for the suggestion.